backend/kernelCI_app/views/buildTestsView.py

```diff
@@ -26,6 +29,13 @@ def get(self, request, build_id: str) -> Response:
                 status=HTTPStatus.OK,
             )
 
+        if result[0]["build__valid"] is False:
```

Comment:
check if result[0] exists

Reply:
I think that the last if (`if not result`) should be enough, shouldn't it?

```python
Python 3.12.7 (main, Nov 18 2024, 15:03:02) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a = []
>>> not a
True
>>> a.append(1)
>>> not a
False
```

Reply:
but I'd still do
```suggestion
        if result[0].get("build__valid") is False:
```

just to be sure